app/models/katello/task_status.rb

```diff
@@ -55,7 +55,7 @@ class Status
           Rails.logger.debug "Unable to report status change" # minor error
           # if logger level is higher than debug logger return false that would cause rollback
           # since this is log only callback we must be sure to return true
-          return true
```

Comment:
Can you also remove the "return true" like 4 lines above this (last line of the begin)